README.md

```diff
@@ -44,6 +44,7 @@ _Related: [Content Management Systems (CMS)](#content-management-systems-cms)_
 - [CKAN](https://ckan.org) - CKAN is a tool for making open data websites. ([Source Code](https://github.com/ckan/ckan)) `AGPL-3.0` `Python`
 - [Collective Access - Providence](https://collectiveaccess.org/) - Highly configurable Web-based framework for management, description, and discovery of digital and physical collections supporting a variety of metadata standards, data types, and media formats. ([Source Code](https://github.com/collectiveaccess/providence)) `GPL-3.0-only` `PHP`
 - [Horahora](https://github.com/horahoradev/horahora) - Video hosting website and video archival manager for Niconico, Bilibili, and Youtube. `MIT` `Go`
+- [Wayback](https://wabarc.eu.org/) - Self-hosted toolkit for snapshot webpage to Internet Archive, archive.today, IPFS, and beyond. ([Demo](https://t.me/wabarc_bot), [Source Code](https://github.com/wabarc/wayback)) `GPL-3.0` `Go`
```

Comment:
- https://wabarc.eu.org returns error messages when trying to archive a page.
- the demo link points to a telegram bot (not self-hosted)

Reply:
@nodiscc Thanks for your reviews!

> https://wabarc.eu.org returns error messages when trying to archive a page.

I tested it and it correctly returned the archived URLs. You are seeing the error message because https://wabarc.eu.org utilized a layer of proxies to time out.

> the demo link points to a telegram bot (not self-hosted)

This telegram bot is also a part of the self-hosted wayback service, which I have also treated as an example.